services/remotecluster/invitation.go

```diff
@@ -52,6 +52,7 @@ func makeConfirmFrame(rc *model.RemoteCluster, siteURL string) (*model.RemoteClu
 	msg := model.RemoteClusterMsg{
 		Id:       model.NewId(),
 		CreateAt: model.GetMillis(),
+		Token:    rc.RemoteToken,
```

Comment:
Here I had to include the Token since it was again causing some more failures during validation. 
Do we need the redundant token in both frame and message structs?

Reply:
Hmm... sorry about that.  Not sure how the feature branch ended up like that.  I'll remove the token from msg when I merge next PR.  FYI, a message can be sent to multiple sites and the tokens are site specific.